src/components/Review/ReviewSection.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react'
+import { Button, Grid, Header, Segment } from 'semantic-ui-react'
+import { SummaryViewWrapper } from '../../formElementPlugins'
+import { TemplateElementCategory } from '../../utils/generated/graphql'
+import { SectionElementStates } from '../../utils/types'
+
+interface ReviewSectionProps {
+  reviewSection: SectionElementStates
+  canEdit: boolean
+}
+
+const ReviewSection: React.FC<ReviewSectionProps> = ({ reviewSection, canEdit }) => {
+  const { section, pages } = reviewSection
+  return (
+    <Segment inverted style={{ backgroundColor: 'WhiteSmoke', margin: '15px 50px 0px' }}>
+      <Header as="h2" content={`${section.title}`} style={{ color: 'Grey' }} />
+      {Object.entries(pages).map(([pageName, elements]) => (
```

Comment:
Getting the same "unique keys" error as yesterday, so should have key attribute on these.

Reply:
Done [2fae4a4](https://github.com/openmsupply/application-manager-web-app/pull/218/commits/2fae4a4968c35b987fab2075ced4fab2107e64ba)